pyartcd/pyartcd/pipelines/release_readiness/checks/blocker_bugs.py

```
@@ -0,0 +1,129 @@
+    try:
+        _, stdout, _ = await exectools.cmd_gather_async(cmd)
+        bugs = json.loads(stdout) if stdout.strip() else []
+    except json.JSONDecodeError:
+        match = re.search(r"Found ([0-9]+) bugs", str(stdout))
+        count = int(match[1]) if match else 0
+        if count > 0:
+            return CheckResult(
+                name="blocker_bugs", status=Status.RED, summary=f"{count} blocker bug(s) (details unavailable) ❌"
+            )
+        bugs = []
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)